src/main/java/org/springframework/hateoas/MediaTypes.java

```diff
@@ -34,4 +35,14 @@
 	 * Public constant media type for {@code application/hal+json}.
 	 */
 	public static final MediaType HAL_JSON = MediaType.valueOf(HAL_JSON_VALUE);
+	
+	/**
+	 * A String equivalent of {@link MediaTypes#HAL_JSON_UTF8}.
+	 */
+	public static final String HAL_JSON_UTF8_VALUE = HAL_JSON_VALUE + ";charset=UTF-8";
```

Comment:
Instead of doing String concatenation, I would write `= "application/hal+json;charset=UTF-8"`, to make this easier to grep and to understand. It would be similar to `org.springframework.http.MediaType`.

Also, I think we should add an underscore to "UTF8" in the constant names: HAL_JSON_UTF_8_VALUE. It's more readable, and it seems to be more conventional when looking at the Spring / Guava / JDK codebases.